### DISCUSSION TITLE
Open Gothic on Mac

### DISCUSSION THREAD

**Discussion by CedricOliverGross** [Try/OpenGothic#254] [Q&A] [answered]
Hello,
Sorry if I didn’t get it but I don’t find any instructions to install open gothic on Mac. Or is it the same as on Linux?
Additionally I got to say that I have an m1 MacBook (Air), are there any special things do do to run it on arm?

**Comment by Try** [comment, marked as answer]
Hi, @CedricOliverGross !
There are instructions in [MacOS/Metal bringup](#142), here they are:
```
# make sure to use x64 compiler, not the arm
export CC=/usr/bin/clang
export CXX=/usr/bin/clang++

cmake -H. -Bbuild -DCMAKE_OSX_ARCHITECTURES=x86_64 -DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo
# set target explicitly: dynamic version of physfs is not really needed, but causing troubles
cmake --build ./build --target Gothic2Notr
```
Also small note from me: sound is not yet working on MacOS, due to issues with openAL

**Reply by Romubay** [reply]
Hello ! 
would it be possible to get a step by step guide, on how to run OpenGothic ? I am, unfortunately, not that technical and have no idea of compiling and so on. I am also using an M1 Mac.

**Reply by Try** [reply]
Build process is described in previous commend. Maybe worth adding: 
aarch64 now supported natively - no need in x64.

For running: once build is finished with success, navigate to build folder:
```
cd /path/to/OpenGothic/out/aarch64/opengothic/
./Gothic2Notr -g /Users/username/steam/gothic2 
```
Sorry, if explanation is not very detail - last time I've done MaxOS bring up was last year - do not remember all details :P
So, fell free to comment, if any difficulties on build step.

**Comment by johnzoki** [comment]
Could somebody tell me how to build for arm64. I already treid it several times, but I'm to inexperienced to find it out myself.

**Reply by Try** [reply]
> I already treid it several times

Do you have an error message or something?

**Comment by johnzoki** [comment]
I done it like described above 
```
export CC=/usr/bin/clang
export CXX=/usr/bin/clang++

cmake -H. -Bbuild -DCMAKE_OSX_ARCHITECTURES=x86_64 -DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo
# set target explicitly: dynamic version of physfs is not really needed, but causing troubles
cmake --build ./build --target Gothic2Notr
```
and replacing "x86_64"  with "arm64" or "aarch64" didnt worked for me. In some instances it told me that clang just works with x86 in some it just failed to build.
But an hour ago found this issue #358 where someone described how he done it and it finally worked!
Would be great to put this in the readme file, so that people who have no idea what they are doing (like me), have the chance to play it natively too.

**Reply by Try** [reply]
The code above is outdated - this is how you cross-compile to x86, using M1 as host.
Today it's way more trivial. Start a new command window and:
```
cmake -H. -Bbuild -DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo
cmake --build ./build --target Gothic2Notr
```
